deploy-manage/deploy/cloud-enterprise/default-system-deployment-versions.md

```diff
@@ -16,6 +16,7 @@ Note that since version 2.7.0, system deployments are automatically upgraded whe
 
 | {{ece}} version | Admin cluster | Logging & Metrics cluster | Security cluster |
 | --- | --- | --- | --- |
+| 4.0.1 | 8.18.0 | 8.18.0 | 8.18.0 |
```

Comment:
The system deployment versions associated to each release, are we sure they haven't changed? Do you know where is that information available? I'd like to learn it for future releases.

It would be nice to add the 3.8.1 in the table as we are also releasing 3.8.1 also, right? Anyway it's not very important.

Reply:
@kilfoyle : it would be great if we get the information about where and how to check the system cluster versions that ECE expects (probably directly in the code or in an internal config file), so we could check this in the future directly ourselves.

The info of that table is really important, it was requested for years by support team and customers :)